android/src/main/java/com/swmansion/rnscreens/SearchBarView.kt

```diff
@@ -15,9 +15,12 @@ class SearchBarView(reactContext: ReactContext?) : ReactViewGroup(reactContext)
     var autoCapitalize: SearchBarAutoCapitalize = SearchBarAutoCapitalize.NONE
     var textColor: Int? = null
     var tintColor: Int? = null
-    var placeholder: String? = null
+    var headerIconColor: Int? = null
+    var textHintColor: Int? = null
+    var placeholder: String? = ""
     var shouldOverrideBackButton: Boolean = true
     var autoFocus: Boolean = false
+    var hintSearchIcon: Boolean = true
```

Comment:
What do you think about changing the name of this variable to `shouldShowHintSearchIcon`?

Reply:
Yes it's a good idea !

Reply:
No, the default value of true does not change the default look, i change the exemple to have the default look